projects/ama-testing/src/e2e/pages/process-properties.card.ts

```diff
@@ -79,6 +81,23 @@ export class ProcessPropertiesCard extends GenericPage {
         await super.click(connectorActionOption);
     }
 
+    async setDecisionTable(dtName: string) {
+        const dtOption = element(by.cssContainingText('.mat-option-text', dtName));
+        await super.click(this.decisionTableSelector);
+        // Workaround:
+        // Click on DT selectbox untill the list of decision tables is populated.
+        let i = 0;
+        try {
+            while (await super.waitForElementToBeInVisible(dtOption, 500)) {
```

Comment:
Wow, this "might be infinite" loops looks a little bit dangerous. Can we say to let it trying only at most X times, like:

```
while (await super.waitForElementToBeInVisible(dtOption, 500) && i < 10) {
    ...
}
```